config/routes.rb

```diff
@@ -1,7 +1,12 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  root 'users#index'
   root 'posts#index'
```

Comment:
@SemyzSerhiy again, 2 roots cannot be used by Rails routes. So please remove the last one